### PR TITLE
Update translation of file run-single-instance-stateful-application.md

### PR DESCRIPTION
**Pull request by Coderhypo**
Hello，

I found the Chinese document very lagging on `run-single-instance-stateful-application.md` 

https://kubernetes.io/zh/docs/tasks/run-application/run-single-instance-stateful-application/ 

And I updated the Chinese document.

### CODE REVIEW

**Comment by chenrui333** [comment]
你可以参看一下最近merge 的PR。
https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Alanguage%2Fzh+is%3Aclosed+zh-trans%3A

看看有没有什么问题。

**Comment by Coderhypo** [comment]
英文原文：https://kubernetes.io/docs/tasks/run-application/run-single-instance-stateful-application/

**Comment by Coderhypo** [comment]
@chenrui333 
Hello，我对 PR 进行了修改，对原有的翻译也做了格式调整
感谢 review

File: content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md [hunk 1]
```diff
@@ -101,77 +81,57 @@ kubectl create -f https://k8s.io/docs/tasks/run-application/gce-volume.yaml
...
-          ----          ------  ------
-          Available     False   MinimumReplicasUnavailable
-          Progressing   True    ReplicaSetUpdated
-        OldReplicaSets:       <none>
-        NewReplicaSet:        mysql-63082529 (1/1 replicas created)
-        Events:
-          FirstSeen    LastSeen    Count    From                SubobjectPath    Type        Reason            Message
-          ---------    --------    -----    ----                -------------    --------    ------            -------
-          33s          33s         1        {deployment-controller }             Normal      ScalingReplicaSet Scaled up replica set mysql-63082529 to 1
+            Conditions:
```
Comment (chenrui333) [inline content/zh/docs/tasks/run-application/run-single-instance-stateful-application.md]: `Conditions`, `OldReplicaSets`, `NewReplicaSet`, on the same level `Pod Template`

**Comment by Coderhypo** [comment]
@chenrui333 Thinks for your review, and I fixed the format.

**Comment by chenrui333** [comment]
/lgtm
/approve

Thanks a lot

**Pull request merged**